### PR TITLE
- Fixed navbar active path/tag/link

### PR DESCRIPTION
**Pull request by Dharmendra205111014**
Hi, 

Few more fixes in continuation of issue #377 and last PR #378 

- Reset loading state when spec changes
- Reset navbar active class from path/tag/link

Signed-off-by: Dharmendra <dharmendra.patel26@gmail.com>

### CODE REVIEW

File: src/rapidoc.js [hunk 1]
```diff
@@ -566,6 +566,19 @@ export default class RapiDoc extends LitElement {
     }
   }
 
+  resetDoc() {
```
Comment (mrin9) [inline src/rapidoc.js:569]: `resetDoc` name is misleading, there are many states in a the doc, it is just reseting the loader and some classes.

IMO, we do not need a separate method for this, just put these statements in `loadSpec()` method

File: src/rapidoc.js [hunk 2]
```diff
@@ -566,6 +566,19 @@ export default class RapiDoc extends LitElement {
...
+    this.resolvedSpec = null;
+    this.loading = true;
+    this.loadFailed = false;
+    this.requestUpdate();
+
+    // Remove the previous active tag
```
Comment (mrin9) [inline src/rapidoc.js:575]: NavBar is available in `read` and `focused` mode only, so check if `renderStyle` is `focused` or `read` then only perform these statements

**Review by mrin9 — CHANGES_REQUESTED** [review]

File: src/rapidoc.js (hunk 2 above)
Comment (Dharmendra205111014) [reply src/rapidoc.js]: Updated PR

**Review by mrin9 — APPROVED** [review]
LGTM

**Review by mrin9 — APPROVED** [review]

**Comment by mrin9** [comment]
Thanks for the PR !!!

**Pull request merged**